model/src/main/kotlin/OrtResult.kt

```diff
@@ -89,7 +88,7 @@ data class OrtResult(
      */
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val labels: Map<String, String> = emptyMap()
-) : ResolutionProvider, PackageConfigurationProvider {
+) : ResolutionProvider {
```

Comment:
Maybe refer to 07d06bbcefdb7e5e183f6eda5fa023d364973f85 in the commit message to confirm that back then we only implemented the `PackageConfigurationProvider` interface for "documentation purposes".